config/locales/cy/errors.yml

```diff
@@ -3,7 +3,7 @@ cy:
     errors:
       messages:
         name: nid yw'n enw dilys
-        blank: 'yn ofynnol '
+        blank: 'yn ofynnol'
```

Comment:
See https://github.com/ministryofjustice/prison-visits-2#welsh-translation – I'll show you and we can update this string tomorrow.